### DISCUSSION TITLE
Bloc adresse internationale non optionnel

### DISCUSSION THREAD

**Discussion by jodionrq** [MTESSDev/FRW#208] [Annonces]
Bonjour,
J'ai un bloc d'adresse internationale que je dois rendre optionnel.
Mon champ "Adresse" est bien optionnel (pas d'astérisque rouge), mais le bloc en soi semble rester obligatoire.
![image](https://github.com/MTESSDev/FRW/assets/124822406/2cdaf582-9377-46a1-8b37-39f42b4b6795)
Est-ce qu'il y a une autre validation à ajouter?

**Comment by j0ber** [comment]
Bonjour,
Il semble que ce soit bien un bug.
Vous pouvez créer un issue.
Nous allons regarder ça.
Merci

**Comment by j0ber** [comment]
Voir issue #209
